scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ash
+
+case "$1" in
+  "agent" )
+    if [[ -z "${NOMAD_SKIP_DOCKER_IMAGE_WARN}" ]]
+    then
+      echo "====================================================================================="
+      echo "!! Running Nomad clients inside Docker containers is not supported.                !!"
+      echo "!! Refer to https://www.nomadproject.io/s/nomad-in-docker for more information.    !!"
```

Comment:
This redirect will only work after https://github.com/hashicorp/web/pull/277 is merged